boards/adafruit/trrs_trinkey/doc/index.rst

```diff
@@ -0,0 +1,155 @@
+.. zephyr:board:: adafruit_trrs_trinkey
+
+Overview
+********
+
+The Adafruit TRRS Trinkey is a small ARM development board with an onboard TRRS
+jack, RGB LED, USB port, and STEMMA QT port.
+
+Hardware
+********
+
+- ATSAMD21E18A ARM Cortex-M0+ processor at 48 MHz
+- 256 KiB flash memory and 32 KiB of RAM
+- Internal trimmed 8 MHz oscillator
+- TRRS jack
+- STEMMA QT I2C Port
+- An RGB NeoPixel LED
+- Native USB port
+- One reset button
+
+Supported Features
+==================
+
+.. zephyr:board-supported-hw::
+
+Connections and IOs
+===================
+
+The `Adafruit TRRS Trinkey Learn site`_ has detailed information about
+the board including `pinouts`_ and the `schematic`_.
+
+System Clock
+============
+
+The SAMD21 MCU is configured to use the 8 MHz internal oscillator
+with the on-chip PLL generating the 48 MHz system clock.  The internal
+APB and GCLK unit are set up in the same way as the upstream Arduino
+libraries.
```

Comment:
would be nice to set your editor less aggressively - looks like it's wrapping at 72 columns but you should do 100. Please apply to the entire file

Reply:
Fixed!